openresty/Dockerfile-alpine

```
@@ -0,0 +1,253 @@
+# Current latest Openresty version is here.
+# Also specify the Nginx core used. (Openresty, adds their own version numbers after the nginx one, so we can't use the same number for both openresty and nginx)
+# Also specify the LUA version used. (Comes with the latest Openresty src bundle.)
+ARG OPENRESTY_VERSION="1.21.4.2" \
+    NGINX_VERSION="1.21.4" \
+    LUA_JIT_VERSION="2.1-20230410"
+
+FROM openresty/openresty:${OPENRESTY_VERSION}-alpine-fat as build
+
+ARG MODSEC_VERSION=3.0.9 \
```

Use value of top-level ARG:

```suggestion
ARG MODSEC_VERSION
```